<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="workArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>731</width>
      <height>631</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X Coordinate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>21</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Coordinate</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_coordinate">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>63</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="y_coordinate">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>20</y>
      <width>63</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="showAxis">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Axes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gridlines">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>70</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Gridlines</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gridOffset">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>110</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="axisWidth">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>150</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>GridOffset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Axis Width</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DDADraw">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>141</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw DDA Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="store">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Store Canvas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Canvas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>240</y>
      <width>181</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Bresenham Line</string>
    </property>
   </widget>
   <widget class="QLabel" name="DDA_TIME">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>190</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Bresenham_TIME">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>240</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="polar_circle">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>340</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>polar circle</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="radius">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>290</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>300</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>radius</string>
    </property>
   </widget>
   <widget class="QLabel" name="polarCircleTime">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>330</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mid_circle">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>380</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>mid circle</string>
    </property>
   </widget>
   <widget class="QLabel" name="midpointCircleTime">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>380</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="major_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>420</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>major</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="major">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>460</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="major_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>420</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>minor</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="minor">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>460</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="polar_ellipse">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>420</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>polar ellipse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bressenham_ellipse">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>bressenham</string>
    </property>
   </widget>
   <widget class="QLabel" name="polarEllipseTime">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>460</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bressenhamEllepseTime">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>500</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Flood_Fill">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Flood Fill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawPolygon">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>540</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>DrawPolygon</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Vertices">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>540</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>610</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Scanline</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>610</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Boundary fill</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
